lib/vnet/daemon/client_darwin.go

```diff
@@ -0,0 +1,197 @@
+// Teleport
+// Copyright (C) 2024 Gravitational, Inc.
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+//go:build darwin
+// +build darwin
+
```

Comment:
FYI: files that end in _darwin are automatically tagged as "// go:build darwin". Same for other platforms.

```suggestion
```

Reply:
Oh thanks. A long time ago I was actually wondering why it needs to be that way, I couldn't find a reason so I assumed it was for some kind of backwards compatibility, but it's not the case I see.

Reply:
The duplicate `//go:` and ``// +` directives were for backwards compat, although I think that has fallen by now. The _platform suffixes have been around for a long time afaik.